apps/prairielearn/src/pages/shared/syncHelpers.js

```diff
@@ -92,7 +92,7 @@ module.exports.pullAndUpdate = async function (locals) {
         const endGitHash = await courseUtil.getCommitHashAsync(locals.course.path);
 
         job.info('Sync git repository to database');
-        const syncResult = await syncFromDisk.syncDiskToSqlAsync(
+        const syncResult = await syncFromDisk.syncDiskToSqlWithLock(
```

Comment:
I would suggest removing the underscore from `_syncDiskToSqlWithLock()` so that the exported name matches the function name (it briefly confused me that we had two slightly different names here).

Reply:
They are (unfortunately) two separate functions - the one with the underscore prefix is a callback-based version that's used internally by *other* functions in this file. Would you mind if I saved that until I can refactor this file to avoid callbacks entirely?